python/rag/healthcare-support-portal/README.md

```diff
@@ -0,0 +1,1661 @@
+```
+
+**Add Galileo for AI observability and monitoring:**
+1. Get your Galileo API key from [app.galileo.ai](https://app.galileo.ai)
+2. Add to `packages/rag/.env`: `GALILEO_API_KEY=your-galileo-key`
+3. Test: `uv run python test_config.py`
+
+At this point, if you've exactly followed the step by steps in the repository, everything should be active except for the Database connection. The Database connection will be configured through the next step.
+
+
+### **Step 3: Launch All Services** ⏱️ *5 minutes*
+
+```bash
+# Start database, backend services, and frontend
+./run_all.sh
+```
+
+**🔍 Watch for Success Indicators:**
+```bash
+✅ PostgreSQL Database (Port 5432)
+✅ Oso Dev Server (Port 8080) 
+✅ Auth Service (Port 8001)
+✅ Patient Service (Port 8002)
+✅ RAG Service (Port 8003)
+✅ Frontend Service (Port 3000)
+
+🌐 Frontend: http://localhost:3000
+```
+
+📋 **What's running:**
+- 🌐 **Web App**: http://localhost:3000 (main interface)
+- 🤖 **RAG API**: http://localhost:8003/docs (upload docs, ask questions)
+- 🔐 **Auth API**: http://localhost:8001/docs (user management)
+- 🏥 **Patient API**: http://localhost:8002/docs (patient records)
+- 🗄️ **Database**: PostgreSQL with vector search
+- ⚖️ **Security**: Oso authorization server
+
+---
+
+## 🎉 **You're Ready!** - Try Your RAG System
+
+### 👥 **Step 4a: Get Demo Users** (Optional)
+
+```bash
+# Health check for all services
+echo "🔍 Running system health check..."
+
+# Check if all services are responding
+echo "Testing Auth Service..."
+curl -s http://localhost:8001/health && echo "✅ Auth Service OK" || echo "❌ Auth Service DOWN"
+
+echo "Testing Patient Service..."
+curl -s http://localhost:8002/health && echo "✅ Patient Service OK" || echo "❌ Patient Service DOWN"
+
+echo "Testing RAG Service..."
+curl -s http://localhost:8003/health && echo "✅ RAG Service OK" || echo "❌ RAG Service DOWN"
+
+echo "Testing Frontend..."
+curl -s http://localhost:3000 > /dev/null && echo "✅ Frontend OK" || echo "❌ Frontend DOWN"
+
+echo "Testing Database..."
+docker exec healthcare-support-portal-db-1 pg_isready && echo "✅ Database OK" || echo "❌ Database DOWN"
+
+echo "🎉 Health check complete!"
+```
+
+### **Step 5: Create Demo Users & Test RAG** ⏱️ *5 minutes*
+
+**5a. Setup Demo Data:**
+```bash
+# Create sample users and data
+uv run python -m common.seed_data
+```
+
+**🔍 Expected Output:**
+```bash
+✅ Created demo users:
+   - Doctor: dr_smith / secure_password
+   - Nurse: nurse_johnson@hospital.com / secure_password
+   - Admin: admin_wilson@hospital.com / secure_password
+✅ Sample patients created
+✅ Demo data seeding complete
+```
+
+**5b. Test the Web Interface:**
+1. 🌐 Open **http://localhost:3000**
+2. 🔑 Login with: `dr_smith` / `secure_password`
+3. 📄 Upload a PDF document (medical guideline, research paper, etc.)
+4. 🤖 Ask a question: *"What are the key recommendations in this document?"*
+5. ✅ Verify you get an AI response with document sources!
+
```

Comment:
I don't see anything in Galileo after asking a question. What log stream does it log to, as this is not set.